src/Core/Silk.NET.Core/Native/ConcurrentDictionaryVTable.cs

```diff
@@ -10,6 +10,10 @@
 
 namespace Silk.NET.Core.Native
 {
+    /// <summary>
+    /// A simple implementation of a <see cref="IVTable" /> using a concurrent dictionary.
+    /// See https://ultz.co.uk/blog/silktouch-slots-vtables/ for an in-depth explanation.
+    /// </summary>
     public class ConcurrentDictionaryVTable : IVTable
```

Comment:
I think providing an alternative implementation of IVTable in Core is incredibly misleading given that we discourage non-generated VTables.

Reply:
True - though I don't think removing it is a good idea either, it can be a useful tool. Maybe deprecating is a good idea, doesn't remove it, but strongly discourages it's use.

Reply:
Will SilkTouch even work with ConcurrentDictionaryVTables these days?

Reply:
actually, I don't think so. Will remove